torch/__init__.py

```diff
@@ -127,10 +127,54 @@ def is_storage(obj):
     return type(obj) in _storage_classes
 
 
+if sys.version_info[0] == 2:
+    _base_str_class = basestring
```

Comment:
Can't you use the functionality provided in https://github.com/pytorch/pytorch/blob/master/torch/_six.py#L29 for this?

Reply:
Thanks for pointing out! I wasn't aware of that :)